lib.php

```diff
@@ -277,8 +300,13 @@ function local_augmented_teacher_send_notloggedin_reminder_message() {
              WHERE rem.messagetype = ?
                AND rem.enabled = 1
                AND rem.deleted = 0";
+    $sql_params = array(0, 1, MESAGE_TYPE_REMINDER);
 
-    $rs = $DB->get_recordset_sql($sql, array(MESAGE_TYPE_NOTLOGGED));
+    $rs = $DB->get_recordset_sql($sql, $sql_params);
+    $user_enrolments = local_augmented_teacher_get_user_enrolments_from_reminders($rs);
+    $rs->close();
```

Comment:
You could use the repo for getting all user enrolments as well: 
$user_enrolments = $user_enrolments_repo->get_all()

Please close the recordset inside the method in the repository.